dist/AppImageBuilder.yml

```
@@ -18,8 +18,7 @@ AppDir:
     exec_args: $@
 
   after_runtime: |
-    echo "X-AppImage-Integrate=false" | tee -a "${TARGET_APPDIR}/org.v1993.evdevhook2.desktop"
-    echo "Test file" | tee -a "${TARGET_APPDIR}/test_file.txt"
+    echo "X-AppImage-Integrate=false" | tee -a "{{APPIMAGE_PATH}}/org.v1993.evdevhook2.desktop"
```

```suggestion
    echo "X-AppImage-Integrate=false" >> "{{APPIMAGE_PATH}}/org.v1993.evdevhook2.desktop"
```

Should work just as well with less log spam.

---

Of course. I just fixed the command.